decidim-proposals/app/views/decidim/proposals/proposals/index.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :meta_title, t(".meta_title", organization: current_organization.name) %>
```

Comment:
We should use the `current_feature.name` (translated) instead of a hardcoded value.